app/src/main/java/com/jvoye/tasky/agenda_detail/presentation/mappers/DateTimeMapper.kt

```diff
@@ -0,0 +1,52 @@
+package com.jvoye.tasky.agenda_detail.presentation.mappers
+
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.format
+import kotlinx.datetime.format.MonthNames
+import kotlinx.datetime.format.char
+import kotlinx.datetime.toLocalDateTime
+import kotlin.time.ExperimentalTime
+import kotlin.time.Instant
+
+fun getItemDetailDateString(localDateTime: LocalDateTime?): String {
```

Comment:
Suggestion: Same idea here as before- you can drop the custom builders and just use `DateTimeFormatter` directly at this call sites 😊.